text/0001-agile-coretime.md

```diff
@@ -0,0 +1,627 @@
+2. The solution SHOULD minimize the barriers to entry in the ecosystem.
+3. The solution SHOULD work well when the Polkadot UC has up to 1,000 cores.
+4. The solution SHOULD work when the number of cores which the Polkadot UC can support changes over time.
+5. The solution SHOULD facilitate the optimal allocation of work to cores of the Polkadot UC, including by facilitating the trade of regular core assignment at various intervals and for various spans.
+6. The solution SHOULD avoid creating additional dependencies on functionality which the Relay-chain need not strictly provide for the delivery of the Polkadot UC.
+
+Furthermore, the design SHOULD be implementable and deployable in a timely fashion; three months from the acceptance of this RFC should not be unreasonable.
+
+## Stakeholders
+
+Primary stakeholder sets are:
+
+- Protocol researchers and developers, largely represented by the Polkadot Fellowship and Parity Technologies' Engineering division.
+- Polkadot Parachain teams both present and future, and their users.
+- Polkadot DOT token holders.
+
+_Socialization:_
+
+The essensials of this proposal were presented at Polkadot Decoded 2023 Copenhagen on the Main Stage. A small amount of socialization at the Parachain Summit preceeded it and some substantial discussion followed it. Parity Ecosystem team is currently soliciting views from ecosystem teams who would be key stakeholders.
+
+## Explanation
+
+### Overview
+
+Upon implementation of this proposal, the parachain-centric slot auctions and associated crowdloans cease. Instead, Coretime on the Polkadot UC is sold by the Polkadot System in two separate formats: *Bulk Coretime* and *Instantaneous Coretime*.
+
+When a Polkadot Core is utilized, we say it is dedicated to a *Task* rather than a "parachain". The Task to which a Core is dedicated may change at every Relay-chain block and while one predominant type of Task is to secure a Cumulus-based blockchain (i.e. a parachain), other types of Tasks are envisioned.
+
+Bulk Coretime is sold periodically on a specialised system chain known as the *Coretime-chain* and allocated in advance of its usage, whereas Instantaneous Coretime is sold on the Relay-chain immediately prior to usage on a block-by-block basis.
+
+This proposal does not fix what should be done with revenue from sales of Coretime and leaves it for a further RFC process.
+
+Owners of Bulk Coretime are tracked on the Coretime-chain and the ownership status and properties of the owned Coretime are exposed over XCM as a non-fungible asset.
+
+At the request of the owner, the Coretime-chain allows a single Bulk Coretime asset, known as a *Region*, to be used in various ways including transferal to another owner, allocated to a particular task (e.g. a parachain) or placed in the Instantaneous Coretime Pool. Regions can also be split out, either into non-overlapping sub-spans or exactly-overlapping spans with less regularity.
```

Comment:
In essence, everything is bulk core time at first. Then, a subset is scheduled to the instantaneous market by the owner (and the owner could be the system itself).

Reply:
Yes, or other Coretime marketplaces that manage allocations on the broker chain